GliaWidgets/Component/Button/Action/ActionButtonStyle.swift

```diff
@@ -62,7 +62,7 @@ public struct ActionButtonStyle: Equatable {
         titleColor: UIColor,
         textStyle: UIFont.TextStyle = .body,
         backgroundColor: ColorType,
-        cornerRaidus: CGFloat = 4.0,
+        cornerRadius: CGFloat = 4.0,
```

Comment:
Unfortunately we cannot fix the typo without also doing a breaking change, because the integrator can also have a call to the init and the init requiring `cornerRadius` instead of `cornerRaidus` would break their compilation. I suggest that you leave it like it was before with the typo, and create a ticket that addresses this separately.

Reply:
Reverted back [90e93ee](https://github.com/salemove/ios-sdk-widgets/pull/459/commits/90e93eeb82019e934ec4faf362d68afee200bb87)